coin/coins.yml

```diff
@@ -1053,3 +1053,11 @@
   decimals: 18
   blockchain: Ethereum
   isTokenSupported: false
+
+- id: 810180
+  symbol: ZKL
```

Comment:
it should be ETH if i'm not mistaken

https://docs.zklink.io/developer/connecting-to-nova

Reply:
well noticed, I just fixed